src/pages/experiments/[experimentId]/plots-and-tables/volcano/index.jsx

```diff
@@ -257,27 +300,32 @@ const VolcanoPlot = () => {
                   onUpdate={updatePlotWithChanges}
                 />
               </Panel>
-              <Panel header='Markers' key='4'>
-                <PointDesign
-                  config={config}
-                  onUpdate={updatePlotWithChanges}
-                />
+              <Panel header="Markers" key="4">
+                <PointDesign config={config} onUpdate={updatePlotWithChanges} />
               </Panel>
-              <Panel header='Add Labels' key='11'>
+              <Panel header="Add Labels" key="11">
                 <> Display Gene Labels Above (-log10 pvalue) </>
                 <Slider
                   value={config.textThresholdValue}
                   min={0}
                   max={spec.maxNegativeLogpValue + 5}
-                  onChange={(value) => {
-                    useRef(_.throttle(() => updatePlotWithChanges({ textThresholdValue: value }), 20));
+                  onChange={value => {
+                    useRef(
+                      _.throttle(
+                        () =>
+                          updatePlotWithChanges({ textThresholdValue: value }),
+                        20,
+                      ),
+                    );
```

Comment:
This is not correct. `useRef` should be declared in the top of the component, not in the `return` block. Refs get created once when the component renders for the first time and persist across renders. If you use `useRef` in the return block, the ref gets created on each render, which makes it useless.

Do this as you did with other components where you used `onUpdateThrottled`.

Reply:
This is still not resolved as far as I can see.